galpy/potential_src/potential_c_ext/SpiralArmsPotential.c

```diff
@@ -308,28 +365,35 @@ double SpiralArmsPotentialz2deriv(double R, double z, double phi, double t,
     double H = *args++;
     double omega = *args++;
 
-    phi = phi - omega * t;
-
-    double g = gam(R, phi, N, phi_ref, r_ref, tan_alpha);
+    double g = gam(R, phi-omega*t, N, phi_ref, r_ref, tan_alpha);
 
     // Return the second derivative of the potential wrt z (d2Phi / dz2)
     double sum = 0;
     int n;
 
+    double Cn;
+    double Kn;
+    double Bn;
+    double Dn;
+
+    double zKB;
+    double tanh2_zKB;
+
     for (n = 1; n <= nCs; n++) {
-        double Cn = *args++;
-        double Kn = K(R, n, N, sin_alpha);
-        double Bn = B(R, H, n, N, sin_alpha);
-        double Dn = D(R, H, n, N, sin_alpha);
+        Cn = *args++;
+        Kn = K(R, n, N, sin_alpha);
+        Bn = B(R, H, n, N, sin_alpha);
+        Dn = D(R, H, n, N, sin_alpha);
 
-        double zKB = z * Kn / Bn;
-        double tanh2_zKB = tanh(zKB) * tanh(zKB);
+        zKB = z * Kn / Bn;
+        tanh2_zKB = tanh(zKB) * tanh(zKB);
 
         sum += Cn * Kn / Dn * ((tanh2_zKB - 1) / Bn + tanh2_zKB) * cos(n * g) / pow(cosh(zKB), Bn);
     }
 
     return -amp * H * exp(-(R - r_ref) / Rs) * sum;
 }
+//LCOV_EXCL_STOP
 
```

Comment:
Think you need a 
```
//LCOV_EXCL_START
```
before phi2deriv as well (and any second derivative below that is not planar, like Rphideriv).